Library/Homebrew/caveats.rb

```diff
@@ -43,9 +43,11 @@ def keg
 
   def keg_only_text
     return unless f.keg_only?
+    reason = ",\nbecause #{f.keg_only_reason}"
 
-    s = "This formula is keg-only, which means it was not symlinked into #{HOMEBREW_PREFIX}."
-    s << "\n\n#{f.keg_only_reason}\n"
+    s = <<-EOS.undent
+      This formula is keg-only, which means it was not symlinked into #{HOMEBREW_PREFIX}#{reason}
```

Comment:
Yeah, sort of intentional due to perfectly understandable style inconsistencies in `core`.

Some `keg_only` explanations in core include the trailing `.`, some don't bother. I couldn't decide whether it was better to have potentially zero `.` or multiple `.` here 😓.

Another issue is that some `keg_only` explanations start with a capital letter, whereas most don't, but with this change the capital letter looks a little more awkward.

I think it's a positive change in general but deciding what to do with the style inconsistencies might not be a bad idea. There's nothing that I saw running with this locally as-is for several months that made me go _"Ew"_ but that's not a promise nobody will notice the differing `keg_only` style that is slightly more exposed with this change.